app/src/main/kotlin/org/cru/godtools/ui/dashboard/lessons/LessonsLayout.kt

```diff
@@ -29,19 +31,28 @@ internal sealed interface DashboardLessonsEvent {
 @Composable
 @OptIn(ExperimentalFoundationApi::class)
 internal fun LessonsLayout(viewModel: LessonsViewModel = viewModel(), onEvent: (DashboardLessonsEvent) -> Unit = {}) {
-    val lessons by viewModel.lessons.collectAsState()
+    val lessonsWithTranslations by viewModel.lessonsWithTranslation.collectAsState(emptyList())
+    val selectedLanguage by viewModel.selectedLanguage.collectAsState(Language(Locale.getDefault()))
+
     LazyColumn(contentPadding = PaddingValues(16.dp)) {
-        item("header", "header") { LessonsHeader() }
+        item("header", "header") {
+            LessonsHeader()
+            HorizontalDivider(modifier = Modifier.padding(vertical = 12.dp))
+            LessonFilters()
+        }
 
-        items(lessons, { it }, { "lesson" }) { lesson ->
+        items(lessonsWithTranslations, { it }, { "lesson" }) { (lesson, translation) ->
```

Comment:
you need to update the `key` for this, by changing the data model now whenever you change languages, or translations reload it will not reuse the same card and you will get strange placement animations